### PR TITLE
release: add kata-pkgsync to pipeline

### PR DESCRIPTION
**Pull request by jcvenegas**
Add pipeline task to do pkgsync.

Fixes: #585

### CODE REVIEW

File: cmd/kata-pkgsync/config.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Fedora_29         : fedora/29
+    Fedora_28         : fedora/28
+    Fedora_30         : fedora/30
+    xUbuntu_18.04     : ubuntu/bionic
+    xUbuntu_16.04     : ubuntu/xenial
+    SLE_12_SP3        : sles/12.3
+    Debian_9          : debian/stretch
+    Debian_10         : debian/buster
+    CentOS_7          : el/7
+    RHEL_7            :
```
Comment (jodh-intel) [inline cmd/kata-pkgsync/config.yaml]: What happens for these last two? Do we need to remove them until they are fully supported?

File: cmd/kata-pkgsync/config.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            - x86_64
+
+packagecloud:
+    repo: katacontainers/testing
+
+distro-mapping:
+    openSUSE_Leap_15.0: opensuse/15.0
+    openSUSE_Leap_42.3: opensuse/42.3
+    Fedora_29         : fedora/29
+    Fedora_28         : fedora/28
```
Comment (jodh-intel) [inline cmd/kata-pkgsync/config.yaml]: Nit: odd numeric ordering (would be good to sort the entire list alphabetically actually ;)

File: cmd/kata-pkgsync/config.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 SUSE LLC
```
Comment (jodh-intel) [inline cmd/kata-pkgsync/config.yaml]: OOI, do we need another SoB if there are other authors?

File: cmd/kata-pkgsync/config.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+#
+# SPDX-License-Identifier: Apache-2.0
+
+obsprojects:
+    home:katacontainers:releases:
+
+        releases:
+            - master
+            - stable-1.8
```
Comment (jodh-intel) [inline cmd/kata-pkgsync/config.yaml]: I'm guessing we're going to need some documentation regarding maintaining this list.

File: cmd/kata-pkgsync/config.yaml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            - stable-1.7
+
+        architectures:
+            - x86_64
```
Comment (jodh-intel) [inline cmd/kata-pkgsync/config.yaml]: Presumably, we're starting with this but can add others later?

File: .ci/azure/pipelines/release.yml [hunk 6]
```diff
@@ -25,3 +25,15 @@ jobs:
       git config --global user.email "azure-pipeline@kata.io"
       git config --global user.name "azure-pipeline"
     displayName: 'Setup'
+  - bash: |
+          cd cmd/kata-pkgsync
+          make
+          ./kata-pkgsync
```
Comment (devimc) [inline .ci/azure/pipelines/release.yml]: super nit:

```
make -C cmd/kata-pkgsync
./cmd/kata-pkgsync/kata-pkgsync
```

File: cmd/kata-pkgsync/config.yaml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+packagecloud:
+    repo: katacontainers/testing
```
Comment (devimc) [inline cmd/kata-pkgsync/config.yaml]: the name `testing` is correct ?, or should we choose a better one, like `staging` , `stable`, `beta` .. ?

**Pull request closed without merging**